packages/dcos-integration-test/extra/test_dcos_diagnostics.py

```diff
@@ -346,155 +361,122 @@ def _check_diagnostics_bundle_status(dcos_api_session):
             assert required_status_field in properties, 'property {} not found'.format(required_status_field)
 
 
-def _create_bundle(dcos_api_session):
+def _create_bundle(diagnostics: Diagnostics):
     last_datapoint = {
         'time': None,
         'value': 0
     }
 
-    health_url = dcos_api_session.default_url.copy(
-        query='cache=0',
-        path='system/health/v1',
-    )
-
-    diagnostics = Diagnostics(
-        default_url=health_url,
-        masters=dcos_api_session.masters,
-        all_slaves=dcos_api_session.all_slaves,
-        session=dcos_api_session.copy().session,
-    )
-
     create_response = diagnostics.start_diagnostics_job().json()
     diagnostics.wait_for_diagnostics_job(last_datapoint=last_datapoint)
     diagnostics.wait_for_diagnostics_reports()
     bundles = diagnostics.get_diagnostics_reports()
-    assert len(bundles) == 1, 'bundle file not found'
-    assert bundles[0] == create_response['extra']['bundle_name']
+    assert len(bundles) > 0, 'bundle file not found'
```

Comment:
Why is this `>` now? The tests shouldn't change, should they?

Reply:
New API reports two bundles (cluster and node) so I need to change from `== 1` to `> 0`